VaticanBAV/et/BAVet196.xml

```diff
@@ -0,0 +1,308 @@
+                          <note>Written in pencil.</note>
+                        </msItem>
+
+                        <msItem xml:id="p2_i3">
+                          <locus target="#80v" />
+                          <title/>
+                          <textLang mainLang="gez" />
+                          <incipit xml:lang="gez">ሀገርነ፡ ኮነት፡ ከመ፡ ሰማርያ፡ እንዘ፡ በላዕሌሃ፡ ወረደ፡ ነጠብጣበ፡ ዝናመ፡ መዓት፡ ወብሔር፡ ኃዘን፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran" /> አምጣነ፡ ረሐብ፡ ድምፀ፡ ዶርሖ፡ ያነቅሐነ፡ በሌሊት።</incipit>
+                        </msItem>
+                      </msContents>
+
+                      <physDesc>
+                        <objectDesc form="Codex">
+                          <supportDesc>
+                            <support>
+                              <material key="parchment" />
+                            </support>
+                            <extent>
+                              <measure unit="leaf">26</measure>
+                              <dimensions type="outer" unit="mm">
+                                <width>98</width>
+                                <height>66</height>
+                              </dimensions>
+                            </extent>
+                            <collation>
+                              <list>
+                                <item xml:id="q9" n="8">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="55r" to="64v" />
+                                </item>
+                                <item xml:id="q10" n="9">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="65r" to="74v" />
+                                </item>
+                                <item xml:id="q11" n="10">
+                                  <dim unit="leaf">6</dim>
+                                  <locus from="75r" to="80v" />
+                                </item>
+                              </list>
+                            </collation>
+                          </supportDesc>
+
+                          <layoutDesc>
+                            <layout writtenLines="15" />
+                          </layoutDesc>
+                        </objectDesc>
+                      </physDesc>
+
+                    </msPart>
+
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <particDesc>
+               <listPerson>
+                 <person>
+                    <persName ref="PRS13202Gabra" role="owner"/><!-- ff. 13v 24v -->
```

Comment:
The information on the persons and the folia on which they are attested would be best encoded as an extra in the respective msPart